go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	google.golang.org/grpc v1.35.0
 	google.golang.org/protobuf v1.25.0
 	gopkg.in/yaml.v2 v2.4.0
-	honnef.co/go/tools v0.0.1-2020.1.6
+	honnef.co/go/tools v0.2.0-0.dev
```

Comment:
Not sure why the bot is bumping to `v0.2.0-0.dev` instead of `v0.1.1`. :thinking: